config.cr

```diff
@@ -21,14 +21,10 @@ Lucky::StaticFileHandler.configure do |settings|
   settings.hide_from_logs = true
 end
 
-Lucky::SessionCookie.configure do |settings|
+Lucky::Session.configure do |settings|
   settings.key = "_authentic_session"
 end
 
-Lucky::CookieJar.configure do |settings|
-  settings.default_expiration = 1.year
```

Comment:
Is this captured as the `permanent` method on cookie now?

Reply:
I was reading up on this and it looks like there doesn't have to be an expiration. By default a cookie will expire when the browser closes. I think that is a reasonable default, but if a user wants something longer someone could use `expires`, `permanent` or set a default with `on_set`